builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -285,6 +285,14 @@ def unDeployService(stage) {
 				env_key = "PROD"
 			}
 			def envBucketKey = "${env_key}${configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX}"
+
+			def apiRootFolder = getApiDocsFolder(stage) //aws s3 rm s3://bucket-name --recursive
+			//empty bucket before deleting it. //
+			try {
+			    sh "aws s3 rm s3://${service_config['event_source_s3']}/ --recursive"
```

Comment:
check for existence of items in the bucket first and only then remove

Reply:
These are the steps followed

- Check config for S3 bucket
- if yes then check is there any content - recursive
- if yes then empty bucket
- then sls delete